app/routes.js

```diff
@@ -140,6 +142,8 @@ export default config => {
       />
       <Route
         path="/tietoja-palvelusta"
+        /* eslint-disable-next-line react/jsx-no-duplicate-props */
+        path={translations.de['about-service-url']}
```

Comment:
Instead of silencing eslint, why not just remove the path assignment above? Besides this, I don't see a need to access the translations if we hard-code the route anyway...

Reply:
If the first path is removed, the other configurations' service pages won't work (which do not cause error in our work but I tried not to mess up anything).
In order to create a more generic code, that would be great if the 'de' part could be substituted by the actual locale (and the translations.js had the displayed path) but I did not find the solution for that yet... This is only a half-way solution, a try-out.